dashboard/app/views/levels/editors/_applab.html.haml

```diff
@@ -70,7 +70,7 @@
           { "entries": ["function1", "function2"], "matchType": "exact", "minTimes": 1, "message": "You need to call both functions." },
           { "entries": ["[forTest]", "moveTo", "randomNumber", "randomNumber"], "matchType": "inexact", "minTimes": 20, "message": "You need to call moveTo in a for loop." }
         ]
-  = f.text_area 'log_conditions', placeholder: 'Insert JSON Data', rows: 4, value: @level.log_conditions ? JSON.pretty_generate(@level.log_conditions) : ''
+  = f.text_area 'log_conditions', placeholder: 'Insert JSON Data', rows: 4, value: @level.log_conditions or ''
```

Comment:
Re: `||` / `or`, don't want to thrash all of the other editors which use the same pattern until they are all changed to pretty_generate.

Re: leaving `pretty_generate` and making it handle the error / testing that it works properly, that is now being tracked in Axosoft and will include making the other fields pretty generate & single point of truthing the logic as well. May get to that if DotD duties slow down.